guides/theme-quality-assurance.md

```diff
@@ -0,0 +1,295 @@
+
+Much of this testing and QA process can be automated. That which can be tested should be. However, there is extreme value in the human eye and all final tests should be done and verified by a **real** human being.
+
+##### Table of Contents
+{:.no_toc}
+
+* An unordered list to be replaced with the ToC, excluding the "##### Table of Contents" from above
+{:toc levels=1}
+
+---
+
+## Testing Environments
+
+These tests should be verified and ran in the following browsing environments or size/widths:
+
+### Desktop
+
+* Windows
+  * Firefox 60+
+  * Chrome C66+
+  * IE 11+
+  * Edge 16+
+  * Opera 53+
+* Mac
+  * Safari 10+
+  * Chrome C66+
+  * Firefox 60+
+  * Opera 53+
+
+### Tablet
+
+* Android
+  * Current Galaxy
+  * Marketshare Winner
+* iOS
+  * Current iPad Pro
+  * Current iPad
+
+### Mobile
+
+* Android
+  * Current Galaxy
+  * Current Pixel
+  * Marketshare Winner
+* iOS
+  * Current Large
+  * Current Small
+  * Marketshare Winner
+
+---
+
+## Theme Sections
+
+### Homepage/Overall
+
+* Homepage is responsive and behaves normally and beautifully at phone, tablet and desktop widths. Measure at varying heights and widths. View window should not obscure any tap targets.
+* Homepage products link properly to their products
+* Homepage navigation is present and consistent throughout the site
+* Client’s brand colors are present and consistent throughout the site
+* Typography is branded and in client’s choice of brand font
+* Social, Meta tags and SEO tags are present and well formed
+* Pixels are present and firing (page view, lead if mail capture)
+* Google Analytics is present and tracking
+* Tap/Click targets are large enough to be clicked properly
+* Body copy is engaging, present and branded to the client
+* Animations are present, but lightweight and functional (enhance, don’t rely)
+* Where Javascript and animation has been deemed necessary, an elegant & workable fallback is offered to support legacy browsers.
+
+### Navigation
+
+* Navigation is responsive and allows for site traversal on mobile and desktop
+* Navigation is clear and links are well formed
+* Navigation leads to the expected locations
+* Navigation hover angles follow Amazon hover convention if mega-menu
+* Tap/Click targets are large enough to be clicked properly
+
+### Footer
+
+* Footer behaves properly on mobile & desktop
+* Social Links are present, branded and functioning
+* Footer links are well formed
+* Footer has copyright present and some form of trust enhancer
+* Tap/Click targets are large enough to be clicked properly
+* Footer links include refs to Privacy Policy and T&C
+
+### Collection
+
+* Collection page is responsive and behaves normally and beautifully at phone, tablet and desktop widths. Measure at varying heights and widths. View window should not obscure any tap targets.
+* Filtering capabilities work across devices and are friendly to interact with
+* Breadcrumbs (if selected are present and functioning)
```

Comment:
To keep things consistent, I would recommend matching this sentence structure to that of the sentence structure under the "Product" heading: "Breadcrumbs (if selected) are present and functioning". Or vice versa depending on which you prefer.